### DISCUSSION TITLE
Are V2 docs published?

### DISCUSSION THREAD

**Discussion by antonagestam** [pydantic/pydantic#5176] [Pydantic V2]
Hi there,

Wanted to check whether latest documentation for V2 is available somewhere, i.e. for `main`? Is this something we want to set up otherwise?

**Comment by samuelcolvin** [comment]
Well the docs from main are published temporarily using smokeshow, but they're pretty out of date. The plan is to work on the V2 docs once we have an alpha released.

**Reply by antonagestam** [reply]
Got it, thanks! I'm keen to start playing with the new facilities for custom types 😊

**Reply by samuelcolvin** [reply]
Good examples on `Color`, `PaymentCard`, and `ImportString`.
